modules/scheduler/batch-job-template/templates/batch-job-base.yaml.tftpl

```diff
@@ -5,8 +5,10 @@ taskGroups:
         - barrier:
             name: "wait-for-node-startup"
         %{~ endif ~}
+        %{~ for runnable in runnables ~}
```

Comment:
If synchronized is turned on, I think we should add a barrier between every runnable. WDYT?

Reply:
Ah, you are right. Will fix.

Reply:
I changed this so we get a "wait-for-script-to-finish" (instead of wait-for-workload) after each script when synchronized=true. This is to not imply that each script is the whole "workload".